src/resources/runCommands.txt

```diff
@@ -1,14 +1,14 @@
-// Download spark
-wget https://archive.apache.org/dist/spark/spark-2.4.8/spark-2.4.8-bin-hadoop2.6.tgz
-
-// Increase driver memory
---driver-memory 8G
+// Download dependencies
+curl -OL https://downloads.datastax.com/dsbulk/dsbulk.tar.gz
+curl -OL https://downloads.datastax.com/enterprise/cqlsh-astra.tar.gz
+wget https://archive.apache.org/dist/spark/spark-2.4.8/spark-2.4.8-bin-hadoop2.7.tgz
```

Comment:
small nit: why we use wget instead of curl here?

Reply:
@weideng1 This file is not used, its just reference commands & we have both the formats (curl & wget)